app/Tournament.js

```diff
@@ -0,0 +1,11 @@
+
+export class Tournament{
+  constructor(Players){
+    this._Players=Players.sort(function(a,b){return a.rank-b.rank}).slice(0,8);
```

Comment:
Avoid horizontal density. Use one line per method call.

Regarding slice(), you're silencing bugs. If you receive more than 8 items, throw. If you receive less, throw.